src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -5,12 +5,12 @@ import {
 } from 'react-beautiful-dnd';
 import { FC, useContext } from 'react';
 import { draggableProps } from '@/interfaces/availabilityPanel.type';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/components/constants/display-sections';
+import { useGetUsersByUsernameQuery } from '@/app/services/usersApi';
```

Comment:
Let us maintain the import order.
Component import should come after external library import ie ```classnames```

Reply:
done